### PR TITLE
Remove experimental parts from documentation

### PR DESCRIPTION
**Pull request by enikao**
I think having experimental parts in the (public) documentation is confusing.

### CODE REVIEW

**Comment by enikao** [comment]
Based on #137

File: lioncore/m3-goal-and-scope.adoc [hunk 1]
```diff
@@ -2,20 +2,18 @@
...
 
-Thera are multiple goals that we can think of.
+There are multiple goals that we can think of.
 Two of the main goals that we can distinguish are as follows:
 
 === Goal 1: Data Interchange Meta-Metamodel
 The goal of the meta-metamodel is to define a meta-language with which to conveniently describe the structure of the model data that is produced and consumed by the LionWeb API.
-This structure is described using node types (concepts), node properties (featues) and node relationships (containment and references).
-w
+This structure is described using node types (concepts), node properties (features) and node relationships (containment and references).
```
Comment (ftomassetti) [inline lioncore/m3-goal-and-scope.adoc:16]: Should we call these links?

File: lioncore/metametamodel.adoc [hunk 2]
```diff
@@ -223,53 +225,11 @@ An EnumerationLiteral is similar to Ecore’s `EEnumLiteral` and to MPS’ `Enum
 .Characteristics
 An EnumerationLiteral has a <<NamespacedEntity.name, `name`>> and an <<NamespacedEntity.key, `key`>>.
 
-An Enumeration is a <<NamespacedEntity>> (as it has an identity and it is contained in the namespace of its <<Enumeration>>).
+An Enumeration is a <<NamespacedEntity>> (as it has an identity, and it is contained in the namespace of its <<Enumeration>>).
 
 .Constraints
 TBD
 
-[[Typedef]]
```
Comment (ftomassetti) [inline lioncore/metametamodel.adoc:231]: Maybe we could open two issues, one for typedef and one for annotation and copy the text regarding annotation in the corresponding issue

File: lioncore/metametamodel.adoc [hunk 3]
```diff
@@ -693,11 +639,35 @@ Uniqueness of id-space ids is out of scope of LIonWeb specification.
...
+
+From an instance, we refer to its defining language element by the language element's <<NamespacedEntity.key, key>>.
+This includes references to
+
+* <<Language>> usage
+* <<Concept>> instance
+* <<Property>> assignment
+* <<Containment>> assignment
+* <<Reference>> assignment
+* <<EnumerationLiteral>> value
```
Comment (ftomassetti) [inline lioncore/metametamodel.adoc:665]: What about PrimitiveTypes?

**Review by ftomassetti — APPROVED** [review]
+1, I just added a couple of comments

File: lioncore/metametamodel.adoc (hunk 2 above)
Comment (enikao) [reply lioncore/metametamodel.adoc:231]: Typedefs: #20 
Annotations: #13

File: lioncore/m3-goal-and-scope.adoc (hunk 1 above)
Comment (enikao) [reply lioncore/m3-goal-and-scope.adoc]: fixed

File: lioncore/metametamodel.adoc (hunk 3 above)
Comment (enikao) [reply lioncore/metametamodel.adoc:669]: I think we refer to them by id, as they are on the same metalevel, and are part of stdlib (not a metamodel). Correct?

I clarified that in the text.

Comment (enikao) [reply lioncore/metametamodel.adoc:669]: Added a todo, so we clarify this later.

**Pull request merged**